sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -2986,7 +2986,7 @@ var setParentClass = function(child, parent) {
             var size = reference.rows * reference.cols;
             for (var i = 0; i < reference.rows; i++)
                 for (var j = 0; j < reference.cols; j++)
-                    ret.set(i, j, output[size * index + j * reference.cols + i]);
+                    ret.set(i, j, output[size * index + j * reference.rows + i]);
```

Comment:
This small function set() sets the value from output array of builtin function in shader through transform feedback. It works in this way: ret[i, j] = output[size * index + j * reference.cols + i]; 
Considering that reference.rows = 2, reference.cols = 3, and index = 0 (get the first result for outerproduct(vec2, vec3). For the original implementation, Then the first result of mat3x2 would be: 
ret[0, 0] = output[0], 
ret[0, 1] = output[3], 
ret[0, 2] = output[6]
ret[1, 0] = output[1], 
ret[1. 1] = output[4], 
ret[1, 2] = output[7]. 
They are not correct, output[6] and output[7] are out of output[0] ~ output[5]. 

After the patch applied, the result would be:
ret[0, 0] = output[0], 
ret[0, 1] = output[2], 
ret[0, 2] = output[4]
ret[1, 0] = output[1], 
ret[1. 1] = output[3], 
ret[1, 2] = output[5]. 

If we use 
ret.set[i, j, output[size * index + i * reference.cols + j]); 
All indices for output array are among (0, 5), but it is not logically correct.